verticapy/machine_learning/metrics/regression.py

```diff
@@ -1192,7 +1193,11 @@ def regression_report(
 
     k: int, optional
         Number  of predictors. Used  to compute the adjusted
-        R2.
+        R2
+
+    genSQL: bool, optional
+        If true return the sql that is used to generate the metrics.
```

Comment:
You can use `` between values. For ex:

If ``True`` ...

and you can use shorter lines.